src/main/java/core/mate/academy/service/impl/BulldozerProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.service.impl;
+
+import static core.mate.academy.enums.BulldozerVariant.ARMORED;
+import static core.mate.academy.enums.BulldozerVariant.REMOTE_CONTROLLED;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
```

Comment:
```suggestion
public class BulldozerProducer implements MachineProducer< Bulldozer > {
```

same for other classes